Jenkinsfile

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2018 Red Hat, Inc. Red Hat licenses this file to you under
+ * the Apache License, version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a
+ * copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+def MAVEN_PARAMS = '-B -e -fae -V -Dsurefire.rerunFailingTestsCount=2'
+
+pipeline {
+
+    agent {
+        label 'checkin'
+    }
+
+    tools {
+        jdk 'jdk8u112'
```

Comment:
maybe worth using a newer version?
or it is to be aligned with all Fuse builds? In this case don't we have a variable to use? otherwise we will have to modify all git repositories when we want to update.

Reply:
Taken verbatim from the current build configuration, can be changed to `'jdk8'` to be in line with _most_ builds, any thoughts @lhein?

Reply:
we should use whatever is used in Fuse Pipeline build